src/azure-cli/azure/cli/command_modules/acs/_consts.py

```diff
@@ -86,6 +86,12 @@
 CONST_NODE_IMAGE_UPGRADE_CHANNEL = "node-image"
 CONST_NONE_UPGRADE_CHANNEL = "none"
 
+# consts for node os upgrade channel
+CONST_NODE_OS_CHANNEL_NODE_IMAGE = "NodeImage"
+CONST_NODE_OS_CHANNEL_NONE = "None"
+CONST_NODE_OS_CHANNEL_SECURITY_PATCH = "SecurityPatch"
```

Comment:
Seems the enum SecurityPatch is not defined in stable API, is there any specific reason?

https://github.com/Azure/azure-rest-api-specs/pull/24487

Reply:
Yeah that didn't ga let me rip it out.